faust/transport/_cython/conductor.pyx

```diff
@@ -78,6 +96,18 @@ cdef class ConductorHandler:
         await chan.put(event)
         delivered.add(chan)
 
+    # callback called when the queue is under high pressure/
+    # about to become full.
+    def on_pressure_high(self) -> None:
```

Comment:
I'm not so familiar with Cython, but would this be faster if it was declared a cpdef? Can it be a cdef? I guess not, as it's passed to another function as an argument.